PopPUNK/visualise.py

```diff
@@ -108,7 +117,9 @@ def get_options():
     faGroup.add_argument('--cytoscape', help='Generate network output files for Cytoscape', default=False, action='store_true')
     faGroup.add_argument('--phandango', help='Generate phylogeny and TSV for Phandango visualisation', default=False, action='store_true')
     faGroup.add_argument('--grapetree', help='Generate phylogeny and CSV for grapetree visualisation', default=False, action='store_true')
+    faGroup.add_argument('--mst', help='Calculate a minimum spanning tree', default=False, action='store_true')
     faGroup.add_argument('--rapidnj', help='Path to rapidNJ binary to build NJ tree for Microreact', default='rapidnj')
+    faGroup.add_argument('--no-nj', help='Do no calculate an NJ tree', default=False, action='store_true')
```

Comment:
How about a `--trees` argument which can take values `nj`, `mst` or `both`, and defaults to `nj`?

Reply:
Done in a5f931a.